haystack/document_stores/sql.py

```diff
@@ -8,7 +8,15 @@
 
 import numpy as np
 
-try:
+from haystack.lazy_imports import LazyImport
+from haystack.schema import Document, Label, Answer
+from haystack.document_stores.base import BaseDocumentStore, FilterType
+from haystack.document_stores.filter_utils import LogicalFilterClause
+
+
+with LazyImport(
+    import_error_msg="Failed to import 'sqlalchemy'. Run 'pip install farm-haystack[sql]'. Original error: {exception}"
```

Comment:
I don't particularly like how verbose is this context manager 😛 and was wondering...

The `Failed to import 'sqlalchemy'.` bit can be derived from the exception, and so `Original error: {exception}`, which leaves `Run 'pip install farm-haystack[sql]'` as the only bit that needs customization.

What if we do something like
```
with LazyImport(hint_msg="Run 'pip install farm-haystack[sql]'") as sqlachemy_import:
    ...
```
and compose the rest of the message inside `__exit__`?

Reply:
Sounds good! I'll do that